### PR TITLE
feat: Agricultural production method field

### LINKED ISSUE #634: Landscape Profile: Add the conditional question for the livelihoods question
## Description
If users select ”Crop farming" and/or "Cattle and livestock farming” in the Livelihoods field, 
DISPLAY the production method field.

## Additional context
image

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Added Aggricultural Production methods field

### Checklist
- [x] Corresponding issue has been opened
- [ ] New tests added
- [ ] English strings reviewed and copyedited
- [ ] Strings are localized
- [ ] Verified on mobile
- [ ] Verified on desktop
- [ ] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))

### Related Issues
Fixes #634 

### Verification steps

image

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -50,3 +50,10 @@ export const PLAUSIBLE_DOMAIN = process.env.REACT_APP_PLAUSIBLE_DOMAIN;
 
 export const LANDSCAPE_PROFILE_IMAGE_MAX_SIZE =
   process.env.REACT_APP_LANDSCAPE_PROFILE_IMAGE_MAX_SIZE || 10000000; // 10 MB
+
+export const AGRICULTURAL_PRODUCTION_METHOD_LIVELIHOODS = process.env
+  .REACT_APP_AGRICULTURAL_PRODUCTION_METHOD_LIVELIHOODS || [
+  'Industry',
+  'Fishing',
+  'Crop farming',
+];
```
Comment (paulschreiber) [inline src/config.js:59]: suggestion: pull these from a strings file or use placeholder text, i.e. `LIVELIHOOD_EXAMPLE`

File: src/localization/locales/en-US.json [hunk 2]
```diff
@@ -458,7 +458,10 @@
...
+        "form_profile_agricultural_production_methods": "Agricultural Production Method",
+        "form_profile_agricultural_production_methods_helper_text": "Which agricultural production method(s) do you use in your landscape? Check all that apply",
```
Comment (paulschreiber) [inline src/localization/locales/en-US.json:463]:
```suggestion
        "form_profile_agricultural_production_methods_helper_text": "Which agricultural production methods do you use in your landscape?",
```

Comment (paulschreiber) [reply src/localization/locales/en-US.json:463]: There's no checkboxes/no checking.

File: src/config.js (hunk 1 above)
Comment (josebui) [reply src/config.js:59]: @paulschreiber These are the specific 3 values for livelihoods that if selected trigger the display of the agricultural production method field, how is that related to a placeholder text? or why should this be inside a strings file?

Comment (paulschreiber) [reply src/config.js:59]: Got it. Please leave a comment to that effect.

File: src/config.js [hunk 3]
```diff
@@ -50,3 +50,11 @@ export const PLAUSIBLE_DOMAIN = process.env.REACT_APP_PLAUSIBLE_DOMAIN;
 
 export const LANDSCAPE_PROFILE_IMAGE_MAX_SIZE =
   process.env.REACT_APP_LANDSCAPE_PROFILE_IMAGE_MAX_SIZE || 10000000; // 10 MB
+
+// List of Livelihoods that if selected will show the agricultural production method field
```
Comment (paulschreiber) [inline src/config.js:54]:
```suggestion
// List of Livelihoods that if selected will show the "agricultural production method" field
```

File: src/config.js [hunk 4]
```diff
@@ -50,3 +50,10 @@ export const PLAUSIBLE_DOMAIN = process.env.REACT_APP_PLAUSIBLE_DOMAIN;
...
```
Comment (paulschreiber) [inline src/config.js:54]:
```suggestion
// List of livelihoods that, if selected, will show the agricultural production method field
```

**Review by paulschreiber — APPROVED** [review]

**Pull request merged**